### ISSUE TITLE
A serious problem happened during setup

### ISSUE THREAD

**Issue by jumi94** [processing/processing#5513]
![image](https://user-images.githubusercontent.com/39464324/40281271-af20ece0-5c03-11e8-969a-ca4ffc12badb.png)

I don't know how to solve this, I've installed Processing, I tried to read an instruction from an Arduino board. I had a problem with the Port, when I tried to run again I found this message.

**Comment by jeremydouglass** [comment]
1. What is your operating system / version?
2. Did you extract the zip before running? -- it looks in your screenshot like that might be a compressed zip file, but I can't tell.
3. There is a 32-bit and a 64-bit download -- did you download the one that matches your operating system? Have you tried the 32?
4. Have you looked at these other issues with the same error message?

-  https://github.com/processing/processing/search?q=%22meant+to+be%22&type=Issues

In particular:

-  https://github.com/processing/processing/issues/4929

**Comment by TheOfficialDoM** [comment]
Hi! 
I'm new to processing and I have the same issue.
I downloaded the 64-bit version, and I have a 64-bit OS.
I extracted the zip file, and even tried to put it into "C:/Processing/" (as I read somewhere else) but it still doesn't work
I looked at #4929 and although the error message seems similar it's not the same.
However downloading the 32-bit version seems to solve the problem as mentioned in #5347

**Comment by ZachSundberg** [comment]
Windows newest security update is treating core.jar as a threat.  It won't even upzip it.

**Comment by Jean-no** [comment]
@ZachSundberg  yes but you can first shut down Microsoft security essentials live protection.
